commodore/component/compile.py

```diff
@@ -74,6 +74,8 @@ def compile_component(config: Config, component_path, value_files, search_paths,
 """)
 
         # Fake Argo CD lib
+        # We plug "Fake" argoCD configuration here because we don't actually
+        # want to recompile that stuff each when compiling a single component.
```

Comment:
The reason to fake the library is more that it just wouldn't exist otherwise. You would have to fake it yourself (and provide it with `-J path/to/fake/lib`) because every component relies on this library.

```suggestion
        # We plug "fake" Argo CD library here because every component relies on it
        # and we don't want to provide it every time when compiling a single component.
```

Does that make sense?